src/content/en/2021/privacy.md

```diff
@@ -5,572 +5,541 @@ description: Privacy chapter of the 2021 Web Almanac covering adoption and impac
+Poor security management within a company can have a significant impact on its customers' private data. <a hreflang="en" href="https://haveibeenpwned.com/">HaveIBeenPwned</a> allows users to check whether their email address or phone number was leaked in a data breach. At the time of this writing, HaveIBeenPwned has tracked 562 breaches, leaking 640 million records. In 2020 alone, 40 domains were breached and personal data about millions of users leaked. Breaches that are _sensitive_ in nature refer to the possibility of a negative impact on the user if someone were to find that user's data in the breach. One example of a sensitive breach is "[Carding Mafia](https://www.vice.com/en/article/v7m9jx/credit-card-hacking-forum-gets-hacked-exposing-300000-hackers-accounts)", a platform where stolen credit cards are traded.
 
 <p class="note">Note that 40 breaches in the previous year is a lower bound, since many breaches are only discovered, or made public, several months after they have occurred.</p>
 
 
 {{ figure_markup(
   image="data_breaches_pwned_websites_per_date_and_breach_sensitivity.png",
   caption="Data breaches in the previous year and their sensitivity",
-  description="Chart that shows the number of domains involved in a breach over the course of the previous year and the sentitivity of the data breach.",
+  description="Chart that shows the number of domains involved in a breach over the course of the previous year and the sensitivity of the data breach. For Not Sensitive breaches it was 4 in August 2020, 2 in September, 6 in October, 1 in November, 6 in December, 5 in January 2021, 3 in February, 5 in March, 2 in April 1 in May, 1 in June, and finally 1 in July. For Sensitive breaches it's just 1 in February 2021 and 2 in March 2021.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRfOwcNkLp-mYBkmhDV5AFxl8p0ls9oxFnmmo0WUcAJxjxQqmHjquRZCWj_lNZRyFtX3RdH5T92IESu/pubchart?oid=2029950563&format=interactive",
   sheets_gid="1435927653",
   sql_file="data_breaches_pwned_websites_per_date_and_breach_sensitivity.sql"
   )
 }}
 
+{# TODO - do we have anything to say about this chart? Perhaps that it's a year-round problem and that the recent tail off is probably more indiciative of under reporting as some breaches will not have been discovered and/or made public yet? If we don't have anything to say then should we just drop the chart? #}
```

Comment:
Do we have anything to say about this chart? Perhaps that it's a year-round problem and that the recent tail off is probably more indiciative of under reporting as some breaches will not have been discovered and/or made public yet? If we don't have anything to say then should we just drop the chart?

Reply:
I would be fine with dropping the chart, but adding a few more details to the text (a.o. the number of sensitive breaches).